### ISSUE TITLE
Is There a way to set default settings ?

### ISSUE THREAD

**Issue by lrAlval** [simov/express-admin#90]
default settings like theme or language ?

Thanks in
Advance

**Comment by simov** [comment]
Yes, through Javascript only. Take a look at this [example](https://github.com/simov/express-admin-examples/blob/master/config/custom/static/theme.js) and its [configuration](https://github.com/simov/express-admin-examples/blob/master/config/mysql/custom.json#L89-L101).
